packages/components/src/Toast/Toast.tsx

```diff
@@ -57,9 +57,7 @@ export default function Toast({
           {children}
         </Text>
       </div>
-      {onDismiss && (
-        <CloseButton color={color} onClose={onDismiss} size="32px" />
```

Comment:
2rem is now the default value for `size`, so we don't need to pass it in here anymore.